Resources/public/ts/react/GridComponent.tsx

```diff
@@ -13,7 +13,8 @@ declare var Translator: BazingaTranslator;
 
 
 interface IProps {
-    src: string;
+    src?: string;
+    data?: Promise<GridConfiguration>;
```

Comment:
"Data" sounds more like the data to be displayed in the grid itself. But that is not the case here, is it?